src/Components/LifecycleTable/LifecycleTable.tsx

```diff
@@ -133,12 +133,20 @@ export const LifecycleTable: React.FunctionComponent<LifecycleTableProps> = ({ d
     switch (dataType) {
       case 'appLifecycle':
         return (
-          <Tr>
-            <Th>{APP_LIFECYCLE_COLUMN_NAMES.name}</Th>
-            <Th>{APP_LIFECYCLE_COLUMN_NAMES.release}</Th>
-            <Th>{APP_LIFECYCLE_COLUMN_NAMES.release_date}</Th>
-            <Th>{APP_LIFECYCLE_COLUMN_NAMES.retirement_date}</Th>
-            <Th>{APP_LIFECYCLE_COLUMN_NAMES.systems}</Th>
+          <Tr key={LIFECYCLE_COLUMN_NAMES.name}>
+            <Th sort={getSortParams(0)}>{LIFECYCLE_COLUMN_NAMES.name}</Th>
```

Comment:
I see these showing up, but they do appear to function yet.

Reply:
Yes let me try and add the sorting in again - had some trouble on the first go around